homeassistant/components/xiaomi_miio/humidifier.py

```diff
@@ -264,7 +264,7 @@ async def async_set_humidity(self, humidity: int) -> None:
             self._device.set_mode,
             AirhumidifierOperationMode.Auto,
         ):
-            self._mode = AirhumidifierOperationMode.Auto.name
+            self._mode = AirhumidifierOperationMode.Auto.value
```

Comment:
This will set mode to a lower case string, but available modes is still with a capital?

Reply:
Yes